### ISSUE TITLE
[1.21] [Neoforge] Neoforge instance is being flagged as Forge version 

### ISSUE THREAD

**Issue by ffuentesm** [MehVahdJukaar/JustEnoughEffectDescriptions#54]
![image](https://github.com/MehVahdJukaar/JustEnoughEffectDescriptions/assets/44974827/5eedeb06-51bf-487b-812f-7529e451da79)
I ma using the jeed-1.21-2.2.0.jar version for NeoForge, and the game flags it as a Forge version
I am using Neoforge 21.0.40

**Comment by ffuentesm** [comment]
This is still now working on Neoforge
 using Neoforge 21.0.42
jeed-1.21-2.2.1.jar 
Crash: https://pastebin.com/ptdiR04h

**Comment by ffuentesm** [comment]
New update still crashes 
using Neoforge 21.0.42
jeed-1.21-2.2.2.jar
log: https://pastebin.com/G4ZF7mPC

**Comment by ffuentesm** [comment]
Still not working 
Crash: https://pastebin.com/g5U5gqXu
using Neoforge 21.0.42
jeed-1.21-2.2.3.jar

**Comment by MehVahdJukaar** [comment]
I officially have no clue whats going on. It works for me in dev

**Comment by ffuentesm** [comment]
I am literally just launching my game with just JEI and JEED installed and it keeps crashing with this message 
`Caused by: org.spongepowered.asm.mixin.injection.throwables.InjectionError: Critical injection failure: Callback method captureHoveredEffect(Lnet/minecraft/client/gui/GuiGraphics;IIIIILnet/minecraft/client/renderer/texture/TextureAtlasSprite;Lcom/llamalad7/mixinextras/injector/wrapoperation/Operation;Lcom/llamalad7/mixinextras/sugar/ref/LocalRef;Lcom/llamalad7/mixinextras/sugar/ref/LocalBooleanRef;Lcom/llamalad7/mixinextras/sugar/ref/LocalRef;)V in jeed.mixins.json:EffectsRenderingInventoryScreenMixin from mod jeed failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap jeed-common-refmap.json`

**Comment by MehVahdJukaar** [comment]
yeah i dont even get an error it just explodes without giving anything on the log. This must be something with mixins as its usually the case. I cant do anything (other than removing one featyre and need to wait for support from them

**Comment by MehVahdJukaar** [comment]
try without other mods

**Comment by ffuentesm** [comment]
> try without other mods

I told you that I did. 
I launched the game with just JEI and JEED and I got that Error. 
If I launch the game without JEED, things do not crash.

**Comment by MehVahdJukaar** [comment]
very werid. without other mods i get no error and it just exists.

**Comment by ffuentesm** [comment]
It works now lol, the game launches without a crash
But this causes EMI to fail to load.
Also, if I launch without any other mod, and just JEI and JEED, things work, but the game crashes as soon as I open the inventory 
Log: https://pastebin.com/cmU0390G

**Comment by MehVahdJukaar** [comment]
is this still anissue in newer versions?

**Comment by ffuentesm** [comment]
Yes, is not being flagged as forge any more
